semgrep/tests/e2e/test_rule_parser.py

```diff
@@ -2,7 +2,8 @@
 from subprocess import CalledProcessError
 
 import pytest
-from tests.conftest import TESTS_PATH
+
+from ..conftest import TESTS_PATH
```

Comment:
Instead of monkeypatching `PYTHONPATH` to do importing here and running tests we should use proper Python modules and do relative imports. Hopefully this can also be a step toward avoiding `subprocess` and simply calling the `semgrep_main` functionality we need directly :+1: